pkg/secretutil/censor.go

```diff
@@ -34,20 +34,32 @@ type Censorer interface {
 }
 
 func NewCensorer() *ReloadingCensorer {
+	return NewCensorerWithMinLength(0)
+}
+
+func NewCensorerWithMinLength(minLength int) *ReloadingCensorer {
 	return &ReloadingCensorer{
-		RWMutex:  &sync.RWMutex{},
-		Replacer: bytereplacer.New(),
+		RWMutex:             &sync.RWMutex{},
+		Replacer:            bytereplacer.New(),
+		minimumSecretLength: minLength,
 	}
 }
 
 type ReloadingCensorer struct {
 	*sync.RWMutex
 	*bytereplacer.Replacer
-	largestSecret int
+	largestSecret       int
+	minimumSecretLength int
 }
 
 var _ Censorer = &ReloadingCensorer{}
 
+// generateReplacement creates an appropriate replacement string for a secret.
```

Comment:
Any specific reason this had to be broken out?

Reply:
No reason, lazy refactor from a version this did more, removed